dotCMS/src/integration-test/java/com/dotcms/datagen/ContainerAsFileDataGen.java

```diff
@@ -20,16 +20,32 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class ContainerAsFileDataGen extends AbstractDataGen<FileAssetContainer> {
 
+    private class ContentTypeContent {
+        private ContentType contentType;
+        private String content;
+
+        public ContentTypeContent(ContentType contentType, String content) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'contentType' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)